webrtc.html

```diff
@@ -8466,7 +8466,13 @@ <h3 id="sec.identity-verify-assertion">Verifying Identity Assertions</h3>
           process asynchronously.</p>
           <p>The IdP proxy verifies the identity assertion using whatever means
           necessary. Depending on the authentication protocol this could
-          involve interacting with the IDP server.</p>
+          involve interacting with the IdP server.</p>
+        </li>
+        <li>
+          <p>If the IdP proxy produces an error or returns a promise that does
+          not resolve to a valid
+          <code><a>RTCIdentityValidationResult</a></code> (see <a href=
+          "#sec.idp-error-handling"></a>), then identity validation fails.</p>
```

Comment:
8472 - 8475 is a copy of 8368 - 8371  - could it be moved to the error handling section?